andela/mont-blanc-cfh
# #151991935 Design landing page with effective UI/UX

What does this PR do?
Design a landing page with effective UI/UX

Description of Task to be completed?
Users should be able to see an overview of what the game is about, how it's played and how it can be used to donate to charity and also see call to action buttons that lead them to signup or signin.

How should this be manually tested?
Navigate to https://mont-blanc-cfh.herokuapp.com/ 
              or 
* Git clone this repo.
* Run "npm install" on the command line.
* Run "npm start" on the command line.
* Visit localhost:3001 on browser.

Any background context you want to provide?
None

What are the relevant pivotal tracker stories?
#151991935 Users should see a landing page with effective UI/UX

Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30434146/32194803-b86a7456-bdbb-11e7-82cc-258f371c6dd4.png)
![image](https://user-images.githubusercontent.com/30434146/32194100-a9502e40-bdb9-11e7-8cf0-2d3a483b38df.png)
![image](https://user-images.githubusercontent.com/30434146/32194114-b33bcb1c-bdb9-11e7-8fcc-831fa4e53b71.png)
![image](https://user-images.githubusercontent.com/30434146/32194924-107ff99a-bdbc-11e7-8000-b732d2032686.png)

Questions:
None
